minecraft_servers/pixelpulsecraft/manifest.json

```diff
@@ -5,17 +5,30 @@
   "server_wildcards": [
     "%.PixelPulseCraft.de"

```

Comment:
Not a real wildcard https://i.imgur.com/TiBTlIu.png
Also, please change the IP to lowercase.